runtime/neumann/src/weights/pallet_xcm.rs

```diff
@@ -183,6 +183,16 @@ impl<T: frame_system::Config> pallet_xcm::WeightInfo for WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().reads(4))
 			.saturating_add(T::DbWeight::get().writes(2))
 	}
+	/// Storage: PolkadotXcm XcmExecutionSuspended (r:0 w:1)
+	/// Proof Skipped: PolkadotXcm XcmExecutionSuspended (max_values: Some(1), max_size: None, mode: Measured)
+	fn force_suspension() -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `0`
+		//  Estimated: `0`
+		// Minimum execution time: 2_972_000 picoseconds.
+		Weight::from_parts(3_082_000, 0)
```

Comment:
Yes. We should 

Yes, we need to run a benchmark on a standard machine to generate this value. 
However, this weight value isn't very critical at the moment because it is typically only called by `AdminOrigin`.

https://github.com/paritytech/polkadot/blob/ba42b9ce51d25bdaf52d2c61e0763a6e3da50d25/xcm/pallet-xcm/src/lib.rs#L1131

In kusama, the weight is 3_078_000. 

https://github.com/paritytech/polkadot/blob/ba42b9ce51d25bdaf52d2c61e0763a6e3da50d25/runtime/kusama/src/weights/pallet_xcm.rs#L179